### PR TITLE
Help Center - New dashboard + personal account migration

### PR DESCRIPTION
**Pull request by jayk-u**
Here comes the awaited product update for the shiny new dashboard and the personal account migration recently performed.

You'll find here:
- the product update
- a new help page explaining the migration
- updated help pages with enhanced screenshots reflecting the new dashboard

What is missing:
- product update visual, have to check that point with Yoan first

Thanks!

P.S. added Yoan as a reviewer (specially for screenshots in the help center as we talked about this)

### CODE REVIEW

**Review by scharrier — COMMENTED** [review]
Can we remove the product update from this PR to keep things clean? One PR for the help center, one for the product update. It will simplify the review.

**Comment by jayk-u** [comment]
> Can we remove the product update from this PR to keep things clean? One PR for the help center, one for the product update. It will simplify the review.

My bad, that was initially my intention but I thought it would have been more consistant as I link the help center in the product update.
Is there a simple way to split it on my end, or should I just brutally move the files, push and then re-open a branch?

**Comment by jayk-u** [comment]
@scharrier I've removed the PU part of this PR, which is now solely focusing on the Help Center part. Would you mind reviewing it? Thanks!

File: src/_help/getting-started/upload-your-first-definition.md [hunk 1]
```diff
@@ -15,23 +15,24 @@ Two options are available: using your email address and creating a username, or
...
 
-Pay attention when choosing your username: it will act as a slug to access your API documentation and will appear in the URL (unless you configure a custom domain later). You can change this username/slug later on.
-
 ## Create your first documentation
 
-### Step 1: Create your documentation
+### Step 1: Create your organization
+
+Now that your Bump.sh account is created, you will immediately be prompted to create your first organization.
+An organization allows you to work alone or collaboratively with your API documentation and hubs (a collection of documentation) within the same workspace.
```
Comment (scharrier) [inline src/_help/getting-started/upload-your-first-definition.md:23]: The with + within surprised me here, but maybe it's a mistake from me.

Just to be sure, here is a suggestion:
```suggestion
An organization allows you to work alone or collaboratively on your API documentation and hubs (a collection of documentation) within the same workspace.
```

File: src/_help/organizations/personal-user-account-migration.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Personal user account migration
+---
+
+- TOC
+{:toc}
+
```
Comment (scharrier) [inline src/_help/organizations/personal-user-account-migration.md:7]: I think we should add a notice about the fact that this page only concerns users with an account created before last week. WDYT?

File: src/_help/organizations/personal-user-account-migration.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## Why this change?
+
+Initially, Bump.sh was first designed around individual usage before gradually shifting towards team collaboration. However, it's this latter need that has become widely predominant.
+
+We observed that personal user accounts sometimes caused confusion, especially for users signing up to join an organization. Conversely, some personal projects could grow enough to transition to team collaboration.
+
+## What does this change mean for me?
+
+For our new users, the change mainly occurs during registration, where you'll be asked to create your organization. You'll enjoy all the features of organizations during your free trial. At the end of it, you can choose to continue benefiting from the full potential of organizations or switch to a free plan.
```
Comment (scharrier) [inline src/_help/organizations/personal-user-account-migration.md:20]: I don't think this is useful: we should probably focus on explaining the changes to our impacted users.

File: src/_help/organizations/personal-user-account-migration.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+- A new organization has been created, adopting your slug/username.
+- All your hubs and documentation have been migrated to this organization.
+- Your existing URLs remain the same and still work; we haven't touched them at all.
+
+## What impact should you expect on pricing?
+
+Previously, organizations were a feature reserved for our paid plans. To accommodate this transition, users on a Free plan now have access to an organization, which includes only public access documentation.
+
+All other organization features (such as access management, invitations, collaborative work) remain reserved for our paid plans.
```
Comment (scharrier) [inline src/_help/organizations/personal-user-account-migration.md:35]: Should we add a link to our pricing page?

File: src/_help/publish-documentation/create-and-manage-documentation.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ title: Create and manage documentation
 
 ## Where can I find my documentation?
 
-By default, accessing your dashboard displays your personal account, which lists all your documentation.
+By default, accessing your dashboard displays your organizations, which list all your documentation.
```
Comment (scharrier) [inline src/_help/publish-documentation/create-and-manage-documentation.md:10]: WDYT of simplifying it into?

```suggestion
By default, accessing your dashboard displays all your documentation.
```

File: src/_help/publish-documentation/create-and-manage-documentation.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ Creating documentation starts from your dashboard, using the button of the same
 ![](/images/help/doc-creation.png)
 
 | Name     | The name of the documentation as it will appear to those who have access to it.     |
-| Hub              | Here, you can specify whether you want to link this documentation to an existing hub or not. Hubs are collections of documentation.     |
+| Description              | Add a short description of your API documentation.     |
```
Comment (scharrier) [inline src/_help/publish-documentation/create-and-manage-documentation.md:21]: Nope, we don't ask this in the API creation form.

File: src/_product-updates/2024-01-06-api-keys.md [hunk 7]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (scharrier) [inline src/_product-updates/2024-01-06-api-keys.md:1]: This as already been merged, you should probably rebase your branch.

File: src/_help/publish-documentation/create-and-manage-documentation.md [hunk 8]
```diff
@@ -18,7 +18,7 @@ Creating documentation starts from your dashboard, using the button of the same
...
 | Slug | The slug allows you to define the URL of your documentation, in the format `https://bump.sh/your-username/your-slug`.     |
 | Documentation Access         | Documentation can be either public and accessible to everyone or private, visible only to you and invited members.     |
```
Comment (scharrier) [inline src/_help/publish-documentation/create-and-manage-documentation.md:23]: We are missing several new fields here:
* the type of the API
* the prefered deployment method

**Review by scharrier — COMMENTED** [review]
Great job!

I left several comments, I think we need to iterate a bit on some parts, but it globally LGTM. Good idea to add a page about the personal account removal 👍

**Comment by jayk-u** [comment]
Latest commit should fix it all. Could you please have a look @ChristopheDujarric and/or @grossyoan?

**Review by grossyoan — APPROVED** [review]
Lgtm!

**Pull request merged**